packages/@coorpacademy-player-services/src/test/analytics.send-viewed-media-analytics.js

```diff
@@ -1,22 +1,65 @@
+// @flow strict
+
 import test from 'ava';
 import {sendViewedMediaAnalytics} from '../analytics';
+import type {Resource} from '../types';
+
+type DataEvent = {|
+  event: string,
+  mediaType: string,
+  location: string
+|};
+
+// eslint-disable-next-line no-shadow
+type Window = {
```

Comment:
Please use strict, and this type is used everywhere, can you import it from a shared types file ?